tests/components/aranet/test_sensor.py

```diff
@@ -61,6 +61,13 @@ async def test_sensors(hass: HomeAssistant) -> None:
     assert press_sensor_attrs[ATTR_UNIT_OF_MEASUREMENT] == "hPa"
     assert press_sensor_attrs[ATTR_STATE_CLASS] == "measurement"
 
+    interval_sensor = hass.states.get("sensor.aranet4_12345_update_interval")
```

Comment:
If you want to be able yo test this without removing `entity_registry_enabled_default=False` you can use the `entity_registry_enabled_by_default` fixture:
https://github.com/home-assistant/core/blob/408c613731c6f5174e9d8eea80af69492175edd3/tests/components/airzone/test_sensor.py#L24C26-L24C60